### ISSUE TITLE
when logits is all zero, why in_top_k will return true?

### ISSUE THREAD

**Issue by aling1472** [tensorflow/tensorflow#10489]
x = tf.constant([[2.0,9.0],[7.0,5.0],[0.0,0.0]])
y = tf.constant([1,0,1])
 tf.nn.in_top_k(x,y,1).eval()
Out : array([ True,  True,  True], dtype=bool)
why not output array([ True,  True,  False], dtype=bool)

**Comment by michaelisard** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
